src/main/java/org/generationcp/breeding/manager/containers/GermplasmQuery.java

```diff
@@ -385,4 +397,23 @@ void retrieveGIDsofMatchingGermplasm() {
 		return allGermplasmGids;
 	}
 
+	private Object getGermplasmGid(final Optional<Germplasm> germplasm) {
```

Comment:
Can we always return String instead of Object for the GID?

Reply:
Okay will make the changes

Reply:
Can we extract `getGermplasmGid` and `getGermplasmPreferredName` to a Utility class? So it can be reused in `GermplasmColumnValuesGenerator` class as well.